civihr_default_theme/scss/components/_my-leaves.scss

```diff
@@ -0,0 +1,3 @@
+.button-container{
```

Comment:
This should be in _scss/layout/_page.scss_ or _scss/layout/_my-leaves.scss_ or _scss/pages/_my-leaves.scss_ (whatever you prefer)

Also remember to namespace this selector with the class associated with this specific page